docs/UserGuide.md

```diff
@@ -106,12 +106,11 @@ Adds a pet owner to PawPatrol.
 <div markdown="block" class="alert alert-primary">
 
 **:exclamation: More on NRIC fields:**<br>
-<ul>
-<li>Due to PDPA regulations, only the last 3 digits and last letter of the NRIC will be shown in the list view.
-<ul><li>The first 4 numbers of the NRIC will be replaced with a `X` to protect the privacy of the owner.</li></ul>
-</ul>
+Due to PDPA regulations, only the last 3 digits and last letter of the NRIC will be shown in the list view. 
```

Comment:
We should add an external link to explain PDPA and specifically the relevant part

Reply:
plz review and resolve convo